src/otserv.cpp

```diff
@@ -38,10 +38,12 @@
 #include "databasemanager.h"
 #include "scheduler.h"
 #include "databasetasks.h"
+#include "redis.h"
 
 DatabaseTasks g_databaseTasks;
 Dispatcher g_dispatcher;
 Scheduler g_scheduler;
+Redis *g_redis = nullptr;
```

Comment:
I just looked that up from https://github.com/otland/forgottenserver/blob/master/src/scriptmanager.cpp#L33-L41